### DISCUSSION TITLE
Czy istnieje metoda API do pobierania opinii o produktach?

### DISCUSSION THREAD

**Discussion by javaguru-pl** [allegro/allegro-api#4617] [Suggestion - verification]
Po przeszukaniu dostępnej dokumentacji - wygląda na to iż nie ma takiej metody.
Czy jest możliwość wdrożenia takowej?
Chodzi o to iż niektórzy sprzedawcy posiadają własne produkty z własnymi kodami EAN i chcieli by monitorować opinie o tych produktach.
Chodzi o takie opinie - przykład poniżej:
https://allegro.pl/oferta/smartfon-xiaomi-redmi-note-9-4-128gb-nfc-lte-gps-9339771008#productReviews

**Comment by Lukasz-Zurek** [comment]
Potwierdzam, że nie mamy takiego zasobu. Nie mamy tego również w planach. Jeśli jednak zdecydujemy się na wdrożenie propozycji - poinformujemy o tym fakcie w wątku.

**Comment by ArchprvS** [comment]
Nawiązując do wcześniejszej dyskusji w tym wątku:

Czy jest możliwość wdrożenia w API możliwości pobierania opinii o produktach sprzedawcy w oparciu o ich własne kody EAN lub inne unikalne identyfikatory?

Dziękuję za wszelkie informacje.

**Reply by PrzemyslawLukanowski** [reply]
Dziękujemy za dodatkowy wpis, na tę chwilę nie mamy tego w planach. Jeśli zdecydujemy się na wprowadzenie takiego rozwiązania, poinformujemy w

**Comment by aatanvardoo** [comment]
Temat wraca co jakiś czas, podbijam

**Reply by PrzemyslawLukanowski** [reply]
Na tę chwilę nie mamy w planach udostępniania zasobu do pobierania opinii o produktach.
